FWCore/ParameterSet/python/Types.py

```diff
@@ -1919,6 +1960,8 @@ def testRequired(self):
             p1.foo = dict(a=5)
             self.assertEqual(p1.dumpPython(),'cms.PSet(\n    foo = cms.PSet(\n        a = cms.int32(5)\n    ),\n    allowAnyLabel_=cms.required.PSetTemplate(\n        a = cms.required.int32\n    )\n)')
             self.assertEqual(p1.foo.a.value(), 5)
+            p1 = PSet(anInt = required.int32)
+            self.assertRaises(TypeError, lambda : setattr(p1,'anInt', uint32(2)))
```

Comment:
These assertions could be also written as
```suggestion
            self.assertRaises(TypeError, setattr, p1,'anInt', uint32(2))
```
to avoid the layer of lambda, but I don't have strong feelings to either direction.

Reply:
changed